tslint.json

```diff
@@ -1,20 +1,16 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-        "tslint:recommended",
-        "tslint-no-unused-expression-chai"
-    ],
-    "jsRules": {},
-    "rules": {
-      "prettier": true
-    },
-    "rulesDirectory": [
-      "tslint-plugin-prettier"
-    ],
-    "linterOptions": {
-      "exclude": [
-        "**/node_modules/**",
-        "**/*.js"
-      ]
-    }
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended", "tslint-no-unused-expression-chai"],
+  "jsRules": {},
+  "rules": {
+    "prettier": true,
+    "interface-name": false,
+    "object-literal-sort-keys": false,
+    "arrow-parens": false,
```

Comment:
I like being able to define single-param arrow functions without parens.

Reply:
Personally, I don't like it any more 😄 I like the reasoning behind it in the [Google javascript style guide](https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions), though, I guess Typescript probably makes it harder to mess it up.

... but that does mean there's a huge refactor if we force it with the linter, prettier, or tsc. so maybe a discussion for another day.